woocommerce/woocommerce
# Add code snippet to add or modify states

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woodocs/issues/103

This PR adds the code snippet [Add or modify states](https://woocommerce.com/document/addmodify-states/) to the Woo docs.

### How to test the changes in this Pull Request:

1. Compare the content of the file docs/snippets/add-or-modify-states.md with the content on https://woocommerce.com/document/addmodify-states/.
2. Verify that the new link within docs/snippets/README.md points to the newly added code snippet.
